### DISCUSSION TITLE
How to parse HTML Response

### DISCUSSION THREAD

**Discussion by Tasty213** [postmanlabs/postman-flows#372] [Q&A] [answered]
I'm trying to automate a workflow that involves loading a standard HTML page and getting a token from the result before then using it in other requests. However, I can only see the output from the Send Request block as a reccord/dict and not an HTML object (for use by the XPath block). Does anyone know how you're meant to handle HML with postman flows?

**Comment by samay-v** [comment, marked as answer]
@Tasty213 You can change what a request is parsed as in the Send Request block itself 
![image](https://user-images.githubusercontent.com/28357106/185103167-38e3a5ed-b6c0-4d37-b213-c4ec3c4eaeb8.png)
